widgets/xviewer/org.eclipse.nebula.widgets.xviewer/src/org/eclipse/nebula/widgets/xviewer/action/ColumnMultiEditAction.java

```diff
@@ -59,17 +59,16 @@ public void run() {
          XViewerLib.popup(XViewerText.get("error"), XViewerText.get("error.no_columns.multi_editable")); //$NON-NLS-1$ //$NON-NLS-2$
          return;
       }
-      ListDialogSortable ld = new ListDialogSortable(new XViewerColumnSorter(), xViewer.getTree().getShell());
-      ld.setMessage(XViewerText.get("ColumnMultiEditAction.title")); //$NON-NLS-1$
+      String title = XViewerText.get("ColumnMultiEditAction.title"); //$NON-NLS-1$
+      ListDialogSortableFiltered ld = new ListDialogSortableFiltered(title, "Select an attribute.", new PatternFilter(),

```

Comment:
Should use XViewerText.get(XX) for internationalisation. Please do not forget both German an French Translation
(for French it is "Sélectionner un attribut")